docs/sphinx/source/naming_convention.rst

```diff
@@ -0,0 +1,172 @@
+.. _naming_convention:
+
+.. SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
```

Comment:
Should we have this in every file?

Reply:
yeah, I think it's good, all the files in libbpf have SPDX headers, so yes